tf_files/modules/cdis-aws-common-logging/logging.tf

```diff
@@ -1,12 +1,12 @@
 # We need a bucket so we can upload logs from Elastic Search, logs from the child account, and
 # Kinesis stream logs
 
-resource "aws_s3_bucket" "child_account_bucket" {
-  bucket = "${var.child_name}-logging"
+resource "aws_s3_bucket" "common_logging_bucket" {
+  bucket = "${var.common_name}-logging"
```

Comment:
We need to enforce that bucket names do not include '_' or '.' - take a look at the `local` block in the `cdis-s3-bucket` module:
```
locals {
  clean_bucket_name = "${replace(replace(var.bucket_name, "_", "-"),".", "-")}"
}
```

Reply:
using '-'. I found out the hardway that S3 doesn't like '.' or '_'.

This should be fine now.